src/master/develop/management.md

```diff
@@ -152,6 +152,57 @@ Managing users and databases requires the appropriate privileges. A user with `P
 
 Each database can be configured with a variety of settings. While some values can be changed at any time (though it may require a database reload to take effect), following ones are fixed and cannot be changed: FileSize, MaxKeyLen, MaxValueLen, MaxTxEntries and IndexOptions.MaxNodeSize.
 
+### Configuration options
+
+Following main database options are available:
+
+| Name | Type | Description |
+|------|------|-------------|
+| replicationSettings | Replication Setings | Repliation settings are described below |
+| indexSettings | Index Settings | Index settings are described below |
+| fileSize | Uint32 | maximum file size of files on disk generated by immudb |
+| maxKeyLen | Uint32 | maximum length of keys for entries stored in the database |
+| maxValueLen | Uint32 | maximum length of values for entries stored in the database |
+| maxTxEntries | Uint32 | maximum number of entries inside one transaction |
+| excludeCommitTime | Bool | if set to true, commit time is not added to transaction headers allowing reproducible database state |
+| maxConcurrency | Uint32 | max number of concurrent operations on the db |
+| maxIOConcurrency | Uint32 | |
+| txLogCacheSize | Uint32 | size of transaction log cache |
+| vLogMaxOpenedFiles | Uint32 | maximum number of open files for payload data |
+| txLogMaxOpenedFiles | Uint32 | maximum number of open files for transaction log |
+| commitLogMaxOpenedFiles | Uint32 | maximum number of open files for commit log |
+| writeTxHeaderVersion | Uint32 | transaction header version, used for backwards compatibility |
+| autoload | Bool | if set to false, do not load database on startup |
+
+Replication settings:
+
+| Name | Type | Description |
+|------|------|-------------|
+| replica | Bool | if true, the database is a replica of another one |
+| masterDatabase | String | name of the database to replicate |
+| masterAddress | String | hostname of the master immudb instance |
+| masterPort | Uint32 | tcp port of the master immudb instance |
+| followerUsername | String | username used to connect to the master immudb instance |
+| followerPassword | String | password used to connect to the master immudb instance |
+
+Additional index options:
+
+| Name | Type | Description |
+|------|------|-------------|
+| flushThreshold | Uint32 | threshold in number of entries between automatic flushes |
+| syncThreshold | Uint32 | threshold in number of entries between flushes with sync |
+| cacheSize | Uint32 | sie of btree node cache |
+| maxNodeSize | Uint32 | max size of btree node in bytes |
+| maxActiveSnapshots | Uint32 | max number of active in-memory btree snapshots |
+| renewSnapRootAfter | Uint64 | |
```

Comment:
| renewSnapRootAfter | Uint64 | threshold in time for automated snapshot renewal during data scans |